### PR TITLE
new landscape in LevelPage / new pictures on the GameMenu and a new p…

### PR DESCRIPTION
**Pull request by Cyrildab**
…icture on the targetButton

### CODE REVIEW

File: frontend/src/components/LevelPage/GameMenu/GameMenu.jsx [hunk 1]
```diff
@@ -1,17 +1,25 @@
 import "./GameMenu.scss";
 import React from "react";
+import JurassiqueCharacter from "../../../assets/pictures/AvatarJurassique.png";
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/GameMenu.jsx]: No Franglais in the code

File: frontend/src/components/LevelPage/GameMenu/GameMenu.jsx [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
+import NoteBook from "../../../assets/pictures/NoteBook.png";
 
 function GameMenu() {
   return (
     <nav>
       <img
-        src="./frontend/src/assets/jurassique.png"
+        className="JurassiqueCharacter"
+        src={JurassiqueCharacter}
         alt="un homme de cromagnon"
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/GameMenu/GameMenu.jsx]: If the image is purely decorative, an empty alt is relevant

File: frontend/src/components/LevelPage/LevelPage.scss [hunk 3]
```diff
@@ -1,6 +1,12 @@
...
-  }
+}
+
+h1 {
+  margin-inline: auto;
+}
+
+.JurassiqueTheme {
+  height: 50rem;
+  width: 70rem;
```
Comment (Yoann-Fortin) [inline frontend/src/components/LevelPage/LevelPage.scss]: Mobile First ?

File: frontend/src/components/TargetButton.jsx [hunk 4]
```diff
@@ -1,29 +1,28 @@
 import React, { useState } from "react";
 import "./TargetButton.scss";
+import Pterodactyle from "../assets/pictures/Pterodactyle.png";
 
 function TargetButton() {
   const [popupMessage, setPopupMessage] = useState("");
+  function HandleEvent() {
```
Comment (Yoann-Fortin) [inline frontend/src/components/TargetButton.jsx:7]: Is this popup a modal?

File: frontend/src/components/TargetButton.scss [hunk 5]
```diff
@@ -1,9 +1,10 @@
-$cd-btn: lightgrey;
-$cd-txt: orange;
+$cd-btn: #e7d7bb;
```
Comment (Yoann-Fortin) [inline frontend/src/components/TargetButton.scss:1]: variable is better

**Review by Yoann-Fortin — APPROVED** [review]

**Review by ItsMatDev — APPROVED** [review]

**Pull request merged**